traffic_monitor/experimental/traffic_monitor/manager/datarequest.go

```diff
@@ -198,6 +204,16 @@ func NewCacheStatFilter(params url.Values, cacheTypes map[enum.CacheName]enum.Ca
 	}, nil
 }
 
+// This is the "spirit" of how TM1.0 works; hack to extract a path argument to filter data (/publish/SomeEndpoint/:argument).
+func getPathArgument(path string) string {
+	pathParts := strings.Split(path, "/")
+	if len(pathParts) == 4 {
```

Comment:
I considered making it `>=` but decided against it. I don't have strong feelings so I'll go ahead and make that change.